### PR TITLE
fixes build error

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy [hunk 1]
```diff
@@ -145,7 +145,7 @@ def parse(String description) {
 			result = zwaveEvent(cmd)
 		}
 	}
-	if (result?.name? == 'hail' && hubFirmwareLessThan("000.011.00602")) {
+	if (result?.name?.equals('hail') && hubFirmwareLessThan("000.011.00602")) {
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy:148]: Does this work? A few of the `zwaveEvent` functions return a list.

Comment (greens) [reply devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy:148]: I believe that the `?` after result will mean it gets marked as false?

Comment (greens) [reply devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy:148]: yeah, just verified

Comment (dkirker) [reply devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy:148]: And `result?.name == 'hail'` is what is throwing the original exception? I would expect that to just result in `null == 'hail'` which would be false.

**Review by dkirker — APPROVED** [review]

**Pull request merged**